stix/base.py

```diff
@@ -176,7 +175,7 @@ def to_xml(self, include_namespaces=True, include_schemalocs=False,
 
         namespace_def = ""
         if include_namespaces:
-            xmlns = parser.get_xmlns_str(ns_info.finalized_namespaces)
+            xmlns = nsparser.get_xmlns_str(ns_info.finalized_namespaces)
```

Comment:
I think line 182 needs to be updated as well since it still refers to the `parser` module.